# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux26mm/tiacx.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- linux-2.6.15/drivers/net/wireless/tiacx/Kconfig.vanilla	2006-02-04 16:30:05.310874250 +0100
+++ linux-2.6.15/drivers/net/wireless/tiacx/Kconfig	2006-02-04 16:30:32.876597000 +0100
@@ -18,6 +18,7 @@
 config ACX_PCI
 	bool "TI acx100/acx111 802.11b/g PCI"
 	depends on ACX && PCI
+	default y
 	---help---
 	Include PCI and CardBus support in acx.
 
